docs/how-to/create-custom-image-using-existing-docker-image.md

```diff
@@ -0,0 +1,202 @@
+## How to create customized Docker images using an existing docker image.
+
+**WARNING**
+
+The recommended way of building extensions is to use [Rest APIs](https://api-explorer.alfresco.com/api-explorer). The AMP extension mechanism may be replaced in the future.
+
+#### Prerequisites
+
+* The extension is packaged inside an AMP
+* Docker
+* Access to Docker registry where the new Docker image will be pushed
+
+### Amps that don`t require additional configurations (easy).
+
+**Note:** We are going to use a fork of the [alfresco-bulk-import tool](https://github.com/Epurashu/alfresco-bulk-import), this is compatible with Alfresco 6.0.X(enterprise)/6.0.3(community) and above.
+### Steps:
+* We are going to use the following folder and file structure:
+
+```bash
+alfresco-custom-image                          (Dir)
+│
+└───Dockerfile                                (File)
+│
+└───amps                                       (Dir)
+    │  
+    └───alfresco-bulk-import-*.amp            (File)
+```
+* Clone and download alfresco-bulk-import project.
+
+* Trigger a Maven build from the root folder of the project (SOMEPATH/alfresco-bulk-import).
+
+```bash
+mvn clean install
+```
+
+* Copy SOMEPATH/alfresco-bulk-import/amp/target/alfresco-bulk-import-*.amp under alfresco-custom-image/amps/.
+
+* Create Dockerfile under alfresco-custom-image/
+
+* Add the following lines to the Dockerfile:
+
+```bash
+### We are going to apply amp(s) on the latest community repository image.
```

Comment:
- Shorten comment to: Apply AMP(s) on the latest Community repository image.
- Replace 'amps' with 'AMPs' in comments, and wherever it makes sense.